### PR TITLE
🔨 Switch NetlifyLogs to use useOvermind

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
Follow-up of #2727

Things I did extra:
- Cleanup `useEffect` call
- Export `NetlifyLogs` by a named export  instead of a `default export`

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/3068/commits/e682bf8ff917503b10e8d8632470e7d3d6b897cc is at https://pr3068.build.csb.dev/s/new.

File: packages/app/src/app/pages/common/Modals/NetlifyLogs/index.tsx [hunk 1]
```diff
@@ -1,50 +1,51 @@
...
-    return () => {
-      clearInterval(interval);
-    };
-  }, [url]);
+    const interval = setInterval(async () => {
+      const { logs: fetchedLogs } = await fetch(netlifyLogsUrl).then(data =>
+        data.json()
+      );
+
+      setLogs(fetchedLogs);
```
Comment (christianalfoni) [inline packages/app/src/app/pages/common/Modals/NetlifyLogs/index.tsx]:
```suggestion
      if (fetchedLogs.length) {
        setLogs(fetchedLogs);
      }
```

If we set empty logs, the initial indication that we are waiting is gone. This solves that issue 😄

File: packages/app/src/app/pages/common/Modals/NetlifyLogs/index.tsx [hunk 2]
```diff
@@ -1,50 +1,51 @@
...
+export const NetlifyLogs: FunctionComponent = () => {
   const {
+    actions: { modalClosed },
     state: {
-      deployment: { netlifyLogs: url },
+      deployment: { netlifyLogs: netlifyLogsUrl },
     },
-    actions: { modalClosed },
   } = useOvermind();
+  const [logs, setLogs] = useState(['Contacting Netlify']);
```
Comment (christianalfoni) [inline packages/app/src/app/pages/common/Modals/NetlifyLogs/index.tsx]:
```suggestion
  const [logs, setLogs] = useState(['Waiting for build to start...']);
```

With the improved indication of build not started, below, we should be more explicit about what it is waiting for. It is not waiting for "contact" with Netlify, it is waiting for the build to start

**Comment by SaraVieira** [comment]
awesome! works great!!

**Comment by MichaelDeBoey** [comment]
With pleasure 🙂

**Pull request merged**